libbeat/scripts/generate_template.py

```diff
@@ -114,6 +116,16 @@ def fill_field_properties(field, defaults):
         properties[field["name"]] = {
             "type": "date"
         }
+    elif field.get("type") == "string" and field["name"] == "message":
```

Comment:
or we ca have a string rule like:

<pre>
if field.get("string") == "string": 
  type: "string",
  index: field["index"]
</pre>